### PR TITLE
[notification-types] HTML rendering of markdown formatted message template

### PR DESCRIPTION
**Pull request by pandafy**
Related to #1 

_This is just to show changes which will be introduced after #19 only and it is the only reason this is not for master branch to keep reviewing easy._

### CODE REVIEW

**Comment by pandafy** [comment]
Is doing something like this okay? (Assigning values to non-existent attributes of a model object)
https://github.com/openwisp/openwisp-notifications/blob/fc60d188cb8d5da71dae8adc52743b64f0e667c0/openwisp_notifications/models.py#L46-L50

**Comment by NoumbissiValere** [comment]
@pandafy is this PR ready for review?

**Comment by pandafy** [comment]
Yes, but since it was not opened w.r.t master I didn't marked it for review.

File: README.rst [hunk 1]
```diff
@@ -219,19 +219,21 @@ These properties can be configured for each notification type:
...
     {% extends 'openwisp_notifications/default_message.md' %}
     {% block body %}
-        {{ notification.target }} has malfunctioned. 
+        [{{ notification.target }}]({{ notification.target_link }}) has malfunctioned. 
     {% endblock body %}
 
 .. note::
     You can access all attributes of the notification using ``notification`` variables in your message template as shown above.
+    Additionally attributes ``actor_link``, ``action_link`` and ``target_link`` are also available for providing hyperlinks to
+    respective objects.
```
Comment (NoumbissiValere) [inline README.rst:236]: why did you put this on a newline?

**Pull request closed without merging**